src/battle-log.ts

```diff
@@ -1016,6 +1018,66 @@ export class BattleLog {
 		this.localizeTime);
 	}
 
+	static initYoutubePlayer(
+		id: string, dim: [string, string], videoId: string, time: number | null = null
+	) {
+		const [height, width] = dim;
+		const loadPlayer = () => {
+			const el = $(`div[data-playerid=${id}]`).get(0);
+			if (!el) return;
+			const player = new window.YT.Player(el, {
+				height,
+				width,
+				videoId,
+				playerVars: {
+					'playsinline': 1,
+				},
+				events: {
+					onStateChange: (event: any) => {
+						if (event.data === window.YT.PlayerState.PLAYING) {
+							for (const curPlayer of BattleLog.players) {
+								if (player.psId === curPlayer.psId) continue;
+								curPlayer.pauseVideo?.();
+							}
+						}
+					},
+					onReady: () => {
+						// scroll chat - if it's a page it shouldn't matter much regardless
```

Comment:
PS intentionally only scrolls if it's already at the bottom which this utterly disregards :[

Reply:
OK. Will remove.